Week2/Section2_w2/MySpotify/Song.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Song.h"
+#include <string>
+
+Song::Song()
+{
+	SongTitle = "";
+	ArtistName = "";
+	Duration = 0;
+}
+
+Song::Song(const std::string NewSongTitle, const std::string NewArtistName, const int NewDuration)
+{
+	SongTitle = NewSongTitle;
+	ArtistName = NewArtistName;
+	Duration = NewDuration;
+}
+
+std::string Song::GetFormatedDuration()
+{
+	std::string Minutes = std::to_string(Duration / 100);
+	std::string Seconds = std::to_string(Duration - (Duration / 100) * 100);
```

Comment:
Two comments here:
1. Why are you using 100? Shouldn't it be 60?
2. Seems to work fine, but you get the same result for Seconds if you just do Duration % 60 (after you change the 100 for 60).

Reply:
The way I handle the input was that the user enters the duration of the song like this : 
example (for 3 minutes and 22 seconds)
>> 322 //this is the user input

But I'll change it to the "seconds" input 👌